code/modules/antagonists/heretic/knowledge/flesh_lore.dm

```diff
@@ -49,7 +49,7 @@
 		/obj/effect/decal/cleanable/blood = 1,
 	)
 	result_atoms = list(/obj/item/melee/sickly_blade/flesh)
-	limit = 3 // Bumped up so they can arm up their ghouls too.
+	limit = 30 // Bumped up so they can arm up their ghouls too. //SKYRAT EDIT - ORIGINAL: 3
```

Comment:
honestly, flesh can have the 20, because they need it more then the other two choices.

Reply:
Sure, I'll go with that for now.